src/concrete/ml/sklearn/base.py

```diff
@@ -647,13 +648,24 @@ def compile(
             )
             is_signed = self.input_quantizers[0].is_signed
 
+            # Get the default crypto-parmas to use with TFHE-rs
             crypto_params = json.loads(fhext.get_crypto_params_radix())  # pylint: disable=no-member
+
+            # FIXME: better handle the bitwidth here
```

Comment:
Can we have an issue associated?

Reply:
here it is: https://github.com/zama-ai/concrete-ml-internal/issues/4718